### PR TITLE
Homework 03

### PR DESCRIPTION
**Pull request by Dubouski**

### CODE REVIEW

File: HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_02.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "#{get_d(line)} FROM: #{get_v4?(line)}TO: #{get_r(line)}"
+rescue NoMethodError
+  nil
+end
+
+def process(str)
+  arr = []
+  str.each_line do |line|
+    unless line.downcase.include?('error')
+      arr << res_msg(line) unless res_msg(line).nil?
```
Comment (labs-ruby) [inline HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_02.rb]: Use `Array#map`

File: HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_03.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+require 'time'
+
+def task_3(str)
+  event_time = []
+  if str.is_a?(String)
+    str.each_line do |line|
+      if line.include?('Calling core with action:')
+        date = get_date(line)
+        event_time << Time.parse(date)
```
Comment (labs-ruby) [inline HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_03.rb]: Use `Array#map`

File: HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_02.rb (hunk 1 above)
Comment (Dubouski) [reply HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_02.rb]: Made refactoring code.

File: HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_03.rb (hunk 2 above)
Comment (Dubouski) [reply HW_03/Andrei_Dubouski/andrei_dubouski_hw_03_t_03.rb]: Made Refactoring code.

**Comment by Dubouski** [comment]
Maby I fixed it)

**Pull request merged**